### DISCUSSION TITLE
Vertical Tabs on right-hand side?

### DISCUSSION THREAD

**Discussion by wispwhat** [soulhotel/FF-ULTIMA#113] [Ideas]
First off, Ultima FF is absolutely amazing, I was using the Arch browser in Windows and am very happy to have something more stable in Linux.

It took me a couple of tries to realize that, after setting up the theme the `Hard Way` that it's necessary to move all the files from the FF-ULTIMA folder to the parent directory before proceeding. I'd recommend adding that to the instructions, although other than that they were very very good instructions once I actually followed them correctly.

**Comment by soulhotel** [comment]
https://github.com/soulhotel/FF-ULTIMA/issues/111
